src/content/en/tools/chrome-devtools/index.md

```diff
@@ -19,94 +22,135 @@ Dogfood: Looking for the latest, [Chrome Canary](https://www.google.com/intl/en/
 
-* [Network Panel Basics](/web/tools/chrome-devtools/network-performance/resource-loading)
-* [Understanding Resource Timing](/web/tools/chrome-devtools/network-performance/understanding-resource-timing)
-* [Network Throttling](/web/tools/chrome-devtools/network-performance/network-conditions)
+Optimize page load performance and debug request issues.
+
+* [Get Started](/web/tools/chrome-devtools/network-performance/)
+* [Network Issues Guide](/web/tools/chrome-devtools/network-performance/issues)
+* [Network Panel Reference](/web/tools/chrome-devtools/network-performance/reference)
 
 <div style="clear:both;"></div>
 
-### Timeline
-<img src="images/timeline-panel.png" alt="Timeline Panel" class="attempt-right">
-Use the Timeline to improve the run time performance of your page by recording and exploring the various events that happen during the lifecycle of a site.
+### Performance panel (previously Timeline panel)
+
+<img src="images/panels/performance.png" alt="Timeline Panel" class="attempt-right">
+Improve the runtime performance of your page by recording and exploring the various events that happen during the lifecycle of a site.
+
+<b>Note</b>: In Chrome 57 the Timeline panel was renamed to the Performance panel.
 
 * [How to look at performance](/web/tools/chrome-devtools/evaluate-performance/timeline-tool)
 * [Analyze runtime performance](/web/tools/chrome-devtools/rendering-tools/)
 * [Diagnose Forced Synchronous Layouts](/web/tools/chrome-devtools/rendering-tools/forced-synchronous-layouts)
 
 <div style="clear:both;"></div>
 
-### Profiles
-<img src="images/profiles-panel.png" alt="Profiles Panel" class="attempt-right">
-Use the Profiles panel if you need more information than the Timeline provide, for instance to track down memory leaks.
+### Memory panel (previously Profiles panel)
+
+<img src="images/panels/memory.png" alt="Profiles Panel" class="attempt-right">
+Profile memory usage and track down leaks.
 
+<b>Note</b>: In Chrome 57 the Profiles panel was renamed to the Memory panel.
+
+* [Fix Memory Problems](/web/tools/chrome-devtools/memory-problems/)
 * [JavaScript CPU Profiler](/web/tools/chrome-devtools/rendering-tools/js-execution)
-* [Heap Profiler](/web/tools/chrome-devtools/memory-problems/)
 
 <div style="clear:both;"></div>
 
-### Application
-<img src="images/application-panel.png" alt="Application Panel" class="attempt-right">
-Use the Resources panel to inspect all resources that are loaded, including IndexedDB or Web SQL databases, local and session storage, cookies, Application Cache, images, fonts, and stylesheets.
+### Application panel (previously Resources panel)
+
+<img src="images/panels/application.png" alt="Application Panel" class="attempt-right">
+Inspect all resources that are loaded, including IndexedDB or Web SQL databases, local and session storage, cookies, Application Cache, images, fonts, and stylesheets.
+
+<b>Note</b>: In Chrome 52 the Resources panel was renamed to the Application panel.
```

Comment:
Please use the standard `Note: something` syntax so you get the DevSite styled note stuff.